articles/why-nodejs-good-for-online-store/index.md

```diff
@@ -0,0 +1,38 @@
+
+**Scalability** - Online stores keep growing over short periods, making scalability essential when deciding which platform to use for development. On the other hand, Node.js has an inbuilt mechanism that enables it to scale, and the developer can change the system according to the need.
+
+**Big and Active Community** – Node.js constantly keeps improving due to their vast communities consisting of active and vibrant developers. With their cooperation, a well-documented [framework](https://hackr.io/blog/what-is-frameworks) has been developed, making app development easier and faster. They have also contributed to the release of ready solutions and code snippets. If one encounters a problem, a high chance is the solution already exists from the same community.
+
+**The benefit of the plugins and packages** - Node.js comes with many packages and plugins applied to most apps. That way, the developers do not need to write a program from scratch but can use open source solutions. It significantly reduces the time of development. It also comprises packages for e-commerce.
+
+**Frontend and Backend Language** - Popular frontend frameworks such as [React](https://reactjs.org/tutorial/tutorial.html), [Ember](https://guides.emberjs.com/release/tutorial/part-1/), and [Angular](https://angular.io/tutorial) use Node.js language. Node.js language can create applications that can run on the frontend and backend. The development process is made easy because there is clear communication between front and back end teams since they understand the two [codebases](https://www.techopedia.com/definition/23962/codebase) better. Also, the development team can be smaller and more efficient. In scaling up, recruiting new team members with no much technological diversity will be easier.
+
+**Real-Time Communication and Easy Management** – Real-time interaction with the customer for an e-commerce store is essential. Since Node.js supports frontend and backend development, it is practical to implement a chat-based platform/tool, whereby the interaction with clients will be instant and communication made easy. With the use of the [NoSQL](https://www.mongodb.com/nosql-explained) database that Node.js supports, the management of product listing is simplified, and the e-commerce admin can find it convenient to manage products whether from it is from a single vendor or multivendor platform.
+
+**Data Streaming** – With Node.js, HTTP requests and responses are processed as continuous data streams instead of considered isolated events. It ensures better processing and faster load times. It is essential for e-commerce since data streaming ensures faster uploads and hence better interactions.
```

Comment:
With Node.js, HTTP requests and responses are processed as continuous data streams instead of `considered` isolated events

Remove considered.